Test/scripts/ci-integration.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -x
+
+trap '>&2 echo Error: Command \`$BASH_COMMAND\` on line $LINENO failed with exit code $?' ERR
+
+Test/scripts/install_magento_integrations.sh
+cp Test/scripts/CouponCode.php ../$MAGENTO_DIR
+cp Test/scripts/FreeShipping.php ../$MAGENTO_DIR
+
+# install bolt plugin
+cd ..
+mkdir -p magento/app/code/Bolt/Boltpay
+cp -r project/. magento/app/code/Bolt/Boltpay/
+cd magento
+php bin/magento module:enable Bolt_Boltpay
+
+# # set config
+php bin/magento config:set payment/boltpay/active 1
+php bin/magento config:set payment/boltpay/api_key $BOLT_STAGING_MERCHANT_API_KEY
+php bin/magento config:set payment/boltpay/signing_secret $BOLT_STAGING_MERCHANT_SIGNING_SECRET
+php bin/magento config:set payment/boltpay/publishable_key_checkout $BOLT_STAGING_MERCHANT_PUBLISHABLE_KEY
+
+# install and run ngrok
+wget -O ngrok.zip https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip
```

Comment:
lets not download a random binary and instead save this in our bucket somewhere.